app/src/main/res/layout/new_game_popup.xml

```diff
@@ -37,19 +40,44 @@
             <EditText
                 android:id="@+id/nameGameField"
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
+                android:layout_height="62dp"
                 android:ems="10"
                 android:hint="Enter game name"
                 android:inputType="text" />
 
             <EditText
                 android:id="@+id/maxPlayerCountField"
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
+                android:layout_height="61dp"
                 android:ems="10"
                 android:hint="Enter max player count"
                 android:inputType="number" />
 
+            <EditText
+                android:id="@+id/newGameNumCoins"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:ems="10"
+                android:hint="Enter the number of coins "
+                android:inputType="number" />
+
+            <EditText
+                android:id="@+id/newGameDuration"
+                android:layout_width="match_parent"
+                android:layout_height="78dp"
+                android:ems="10"
+                android:hint="Enter the duration of the game in minutes"
+                android:inputType="numberDecimal" />
+
+            <EditText
+                android:id="@+id/newGameRadius"
```

Comment:
You are missing the autofill hints attribute here

Reply:
Yes